tests/positive/security/tls.go

```diff
@@ -77,6 +87,85 @@ T3Jig2rNmBtpMA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0EAwIDaQAwZgIxAOul
 t3MhI02IONjTDusl2YuCxMgpy2uy0MPkEGUHnUOsxmPSG0gEBCNHyeKVeTaPUwIx
 AKbyaAqbChEy9CvDgyv6qxTYU+eeBImLKS3PH2uW5etc/69V/sDojqpH3hEffsOt
 9g==
+-----END CERTIFICATE-----`)
+
+	// generated via
+	// openssl ecparam -genkey -name secp384r1 -out server.key
+	privateKey1 = []byte(`-----BEGIN EC PARAMETERS-----
```

Comment:
As a separate commit, it might be nice to move these large constants into a single common place, rather than adding to the repetition here.